jazz_environment-event-handler/test/test.js

```diff
@@ -109,12 +113,39 @@ describe('jazz environment handler tests: ', () => {
 			return obj.callback(null, testPayloads.envCreationResponseSuccess, testPayloads.envCreationResponseSuccess.body);
 		});
 
-		let processEachEvent = index.processEachEvent(kinesisPayload.Records[0], configData, authToken);
-		
-		processEachEvent.then((res) => {
+		index.processEachEvent(kinesisPayload.Records[0], configData, authToken)
+		.then((res) => {
 			sinon.assert.calledTwice(requestPromiseStub);
-			requestPromiseStub.restore();
-			expect(res.message).to.include(resMsg);;
+			requestPromiseStub.restore();	
+			expect(res.message).to.include(resMsg);
+		});
+	});
+
+	it('Verify processEachEvent for COMMIT_TEMPLATE event failed', () => {
+		let event = require('./COMMIT_TEMPLATE');
+		const statusCode = testPayloads.apiResponse.statusCode;
+		testPayloads.apiResponse.statusCode = 400;
+		let requestPromiseStub = sinon.stub(request, "Request", (obj) => {
+			return obj.callback(null, testPayloads.apiResponse, testPayloads.apiResponse.body);
+		});
+
+		index.processItem(event.Item, configData, authToken)
+		.catch((res) => {
+			sinon.assert.calledTwice(requestPromiseStub);
+			requestPromiseStub.restore();	
+			expect(res.error).to.include('Error creating stg environment for jazztesting_test-env-oss-3 in  catalog');
```

Comment:
Is there a better way to validate this unit test? Validating on a message that includes test variables hidden in an external file makes this test case fragile. I would just stop at checking if the error message include `Error creating`